### ISSUE TITLE
Compressed JS file corrupted/broken

### ISSUE THREAD

**Issue by jujopalacio** [jazzband/django-pipeline#747]
I've been using django-pipeline==1.7.0 long time ago for my production, testing and review environments on Heroku, using python 2.7.18 and Djando 1.11.

At 04/28/2021 the UI started failing and the console was showing an error on the compressed js file (core.min.js) used as output in our 
PIPELINE:
![image](https://user-images.githubusercontent.com/40274999/117059073-07021f80-ace5-11eb-94e9-a0a781b76570.png)

```
PIPELINE = {
    'JAVASCRIPT': {
        'js_core': {
            'source_filenames': (
                'js/jquery-2.1.4.min.js',
                'js/bootstrap.js',
                'js/inspectlet.js',
                .
                .
                .
                'js/extra.js',
            ),
            'output_filename': 'js/core.min.js',
        }
        },
    }
}
```

We did not upgrade the dependency, we did not modify, update nothing of the statics files included in the `source_filenames`

Something curios is that we had some test environments that was working fine with the compressed file *before* the date I mention things started working weird, I made a diff between the 2 compressed core.min.js and there is this small difference:

![image](https://user-images.githubusercontent.com/40274999/117058914-d5895400-ace4-11eb-8ed2-719846a20eb1.png)

left side is working, right side is not working

Any idea about what could happened?

Thanks

**Comment by jujopalacio** [comment]
UPDATE:

error seems to be a new release of uglify-js I was using version 3.13.4 and now the version released 5 days ago is 3.13.5.

this one is a dependency of yuglify

**Comment by jujopalacio** [comment]
FIX:

I could fix the issue overriding the yuglify dependency using nodejs buildpack **before** python build pack and also using a `npm-shrinkwrap.json` to use uglify dependency 3.13.4
